data/offsets/rec.csv

```diff
@@ -1,21 +1,7 @@
 iso,type,state,price,year,month,source,notes
 national,compliance,,33.94,2021,average,https://www.spglobal.com/market-intelligence/en/news-insights/research/us-renewable-energy-credit-market-size-to-double-to-26-billion-by-2030,
 national,voluntary,,3,2021,average,https://www.spglobal.com/market-intelligence/en/news-insights/research/us-renewable-energy-credit-market-size-to-double-to-26-billion-by-2030,
-caiso,compliance,,70.918,2023,1,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,72.449,2023,2,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,74.49,2023,3,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,81.122,2023,4,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,78.571,2023,5,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,77.041,2023,7,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,72.449,2023,8,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,72.959,2023,10,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,69.898,2023,10,ARB Monthly Average https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,63.776,2023,1,Agrus Media Monthly Index https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,64.286,2023,2,Agrus Media Monthly Index https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,68.367,2023,3,Agrus Media Monthly Index https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,78.061,2023,4,Agrus Media Monthly Index https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,72.959,2023,6,Agrus Media Monthly Index https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
-caiso,compliance,,68.878,2023,11,Agrus Media Monthly Index https://ww2.arb.ca.gov/resources/documents/lcfs-data-dashboard,
+caiso,voluntary,,12.63,2022,average,https://www.sce.com/regulatory/regulatory-information/ferc-Standards-conduct/tariff-books/rates-pricing-choices/renewable-energy-credit,

```

Comment:
Are the units in $/MWh? If so the source has $0.1263/kWh => $126.30/MWh, but it's currently $12.63/MWh

Reply:
Although that was reduced to $31.73/MWh in 2024, so the price seems volatile year-to-year